### ISSUE TITLE
Perforated Backpropagation

### ISSUE THREAD

**Issue by narausch** [pytorch-fdn/ecosystem#13]
### Contact emails
[rorry@perforatedai.com](mailto:rorry@perforatedai.com)

### Project summary
Perforated Backpropagation combines modern neuroscience with modern software to create neural networks.

### Project description
Perforated Backpropagation combines modern neuroscience with modern software to create neural networks that are smarter, smaller, and more accurate with our patent protected technology. Empowering data scientists to achieve unprecedented efficiency and performance by advancing the machine learning they work with, faster.

### Are there any other projects in the PyTorch Ecosystem similar to yours? If, yes, what are they?
There are other researchers looking into the idea of adding dendrites to neural networks, but none are adding them to modern frameworks like PyTorch

### Project repo URL
https://github.com/PerforatedAI/PerforatedAI-API

### Additional repos in scope of the application
No response

### Project license
Free unlimited use licesnse for one iteration, paid license for multiple iterations

### GitHub handles of the project maintainer(s)
RorryB

### Is there a corporate or academic entity backing this project? If so, please provide the name and URL of the entity.
Perforated AI

### Website URL
No response

### Documentation
https://github.com/PerforatedAI/PerforatedAI-API

### How do you build and test the project today (continuous integration)? Please describe.
Currently only one contributor

### Version of PyTorch
No response

### Components of PyTorch
Built on PyTorch. Leverage's PyTorch's register_hook functionality to allow for modifications to standard gradient descent and PyTorch's extreme superiority compared to other frameworks in changing model definitions during runtime.

### How long do you expect to maintain the project?
I plan to continue to maintain this project indefinitely.

### Additional information
No response
